workspaces/global-header/.changeset/grumpy-suns-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@red-hat-developer-hub/backstage-plugin-global-header': major
```

Comment:
There is still things missing. Can we go with patch or minor until we're ready?

Reply:
Will change it to minor. I thought we needed a first version out then we can improve it on top of it as patches.